main.js

```diff
@@ -0,0 +1,108 @@
+//Reassign Squares
+let c0 = document.getElementById('cell-0')
+let c1 = document.getElementById('cell-1')
+let c2 = document.getElementById('cell-2')
+let c3 = document.getElementById('cell-3')
+let c4 = document.getElementById('cell-4')
+let c5 = document.getElementById('cell-5')
+let c6 = document.getElementById('cell-6')
+let c7 = document.getElementById('cell-7')
+let c8 = document.getElementById('cell-8')
+//Boolean value for if there is a winner
+let winner = false
+//Cells Array
+let cells = [c0, c1, c2, c3, c4, c5, c6, c7, c8]
+//Winning Arrays
+let winningconditons = [[c0, c1, c2], [c3, c4, c5], [c6, c7, c8], [c0, c3, c6], [c1, c4, c7], [c2, c5, c8], [c0, c4, c8], [c2, c4, c6]]
+//Players and Status
+let playerX = 'X'
+let playerO = 'O'
+currentPlayer = playerX
+let message = document.getElementById('status')
+//--------------------------------------------------------------------------------------
+//Start Game 1 Player
+let startButton = document.getElementById("start1")
+startButton.addEventListener('click', activateSquare)
+
+function activateSquare() {
+    c0.addEventListener('click', event), c1.addEventListener('click', event), c2.addEventListener('click', event), c3.addEventListener('click', event), c4.addEventListener('click', event), c5.addEventListener('click', event), c6.addEventListener('click', event), c7.addEventListener('click', event), c8.addEventListener('click', event)
+    message.innerHTML = 'Current Player X'
+}
+//Start Game 2 Player
+let startButton2 = document.getElementById("start2")
+startButton2.addEventListener('click', activateSquare2)
```

Comment:
Try and keep all your global variable definitions together, preferably at the top of the file.